### PR TITLE
Added support for KBL version 2.5

### PR DESCRIPTION
**Pull request by takebene**
## Pull Request

- [x] I have checked for similar PRs.
- [ ] I have read the [contributing guidelines](https://github.com/4Soft-de/kbl-model/blob/develop/.github/CONTRIBUTING.md).

### Changes

- [x] Code
- [ ] Documentation
- [ ] Other: 

### Description

Two new modules have been added: "common" and "v25". The "common" module contains the common code of "v24" and "v25".  The "v25" contains the KBL 2.5 model and additional interfaces.

### CODE REVIEW

File: v25/src/test/resources/sample.kbl [hunk 1]
```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<kbl:KBL_container xmlns:kbl="http://www.prostep.org/Car_electric_container/KBL2.3/KBLSchema" id="ID000" version_id="version_id0">
```
[the remark this answers is not in the document]
Comment (RainerGanss) [reply v25/src/test/resources/sample.kbl:2]: same thing with the "real" kbl - it has 2.4 in its name.

File: v25/src/test/java/com/foursoft/harness/kbl/v25/HasPartsTest.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
+package com.foursoft.harness.kbl.v25;
```
Comment (RainerGanss) [inline v25/src/test/java/com/foursoft/harness/kbl/v25/HasPartsTest.java:1]: why is here the license missing?

**Review by RainerGanss — APPROVED** [review]

File: v25/src/test/java/com/foursoft/harness/kbl/v25/HasPartsTest.java (hunk 2 above)
Comment (takebene) [reply v25/src/test/java/com/foursoft/harness/kbl/v25/HasPartsTest.java:26]: i forgot to run "mvn clean install

File: v25/src/test/resources/sample.kbl (hunk 1 above)
[the remark this answers is not in the document]
Comment (takebene) [reply v25/src/test/resources/sample.kbl]: I have adjusted it except for the real KBL. I don't like to adjust that.

**Pull request merged**